src/ngraph/runtime/cpu/pass/cpu_fusion.hpp

```diff
@@ -64,6 +64,8 @@ class ngraph::runtime::cpu::pass::CPUFusion : public ngraph::pass::GraphRewrite
             construct_zero_padded_conv();
             construct_zero_padded_conv_backprop_filters();
             construct_conv_bias_bprop();
+            construct_conv_bias_folded_batch_norm();
```

Comment:
1. Should `construct_conv_bias_folded_batch_norm()` be before `construct_batch_norm_relu()`? If yes, can you please add a comment?

Reply:
Its not required since folded_batch_norm() matches against batchnorm which will come earlier to a relu in the batch_norm_relu() pattern.